chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html

```diff
@@ -3,10 +3,12 @@
     <img src="/assets/images/chucksPickThreeLogo.png" alt="logo" width="50px" height="50px">
   </div>
   <div class="header-img-container">
-    <img src="/assets/images/ChucksPick3HeaderDesktop.png" class="desktopHeader" alt="logo" width="700px"
+    <img src="src/app/assets/images/ChucksPick3HeaderDesktop.png" class="desktopHeader" alt="logo" width="700px"
+         height="100px">
+    <img src="src/app/assets/images/ChucksPick3HeaderTablet.png" class="tabletHeader" alt="logo" width="1000px"
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images